akeneo/pim-community-dev
# RAB-1130: Replace box/spout by openspout/openspout

**Description (for Contributor and Core Developer)**

In this PR, I replaced `box/spout` by `openspout/openspout`. That's the dependency we use to deal with CSV/XLSX file reading. Unfortunately, it is [not maintained anymore](https://github.com/box/spout#-archived-project-) by Box team. A [fork](https://github.com/openspout/openspout#openspout) has been created and is still maintained under `openspout/openspout`.

Manual testing:
I done CSV and XLSX export/import and Tailored Import on envs with box/spout and openspout/openspout, I didn't see any regression on behavior or performance.

**Definition Of Done (for Core Developer only)**

- [ ] Migration & Installer
- [ ] PM Validation (Story)
- [ ] Changelog (maintenance bug fixes)
- [ ] Tech Doc
